From 0ead47041aaa47615c7b78c17680497af05b660a Mon Sep 17 00:00:00 2001
From: Puranjay Mohan <pjy@amazon.com>
Date: Fri, 7 Feb 2025 12:33:03 +0000
Subject: [PATCH] data-new

Signed-off-by: Puranjay Mohan <pjy@amazon.com>
---
 fs/proc/meminfo.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/fs/proc/meminfo.c b/fs/proc/meminfo.c
index 245171d9164b..582150354542 100644
--- a/fs/proc/meminfo.c
+++ b/fs/proc/meminfo.c
@@ -31,6 +31,8 @@ static void show_val_kb(struct seq_file *m, const char *s, unsigned long num)
 	seq_write(m, " kB\n", 4);
 }
 
+static int foo = 5;
+
 static int meminfo_proc_show(struct seq_file *m, void *v)
 {
 	struct sysinfo i;
@@ -157,6 +159,7 @@ static int meminfo_proc_show(struct seq_file *m, void *v)
 	show_val_kb(m, "CmaFree:        ",
 		    global_zone_page_state(NR_FREE_CMA_PAGES));
 #endif
+	seq_printf(m, "kpatch:         %d\n", foo);
 
 #ifdef CONFIG_UNACCEPTED_MEMORY
 	show_val_kb(m, "Unaccepted:     ",
-- 
2.47.1

